packages/components/src/internal/components/domainproperties/samples/SampleTypePropertiesPanel.tsx

```diff
@@ -389,9 +405,9 @@ class SampleTypePropertiesPanelImpl extends React.PureComponent<
                     </Row>
                 )}
 
+                <SectionHeading title="Storage Settings" />
```

Comment:
Can we get into a case where this heading is showing with nothing in the section? i.e. in LKS for community?